### PR TITLE
Rearrange "Help" section and reword "About" to make it more clear

### PR DESCRIPTION
**Pull request by pratyaksh1610**
## Before

WhatsApp Image 2023-02-26 at 8 09 37 PM (1)

## After

WhatsApp Image 2023-02-28 at 3 42 53 PM

- [x] I signed the [CLA](https://musescore.org/en/cla)
- [x] The title of the PR describes the problem it addresses
- [x] Each commit's message describes its purpose and effects, and references the issue it resolves
- [x] If changes are extensive, there is a sequence of easily reviewable commits
- [x] The code in the PR follows [the coding rules](https://github.com/musescore/MuseScore/wiki/CodeGuidelines)
- [x] There are no unnecessary changes
- [x] The code compiles and runs on my machine, preferably after each commit individually
- [x] I created a unit test or vtest to verify the changes I made (if applicable)

### CODE REVIEW

File: src/appshell/internal/applicationuiactions.cpp [hunk 1]
```diff
@@ -54,10 +54,10 @@ const UiActionList ApplicationUiActions::m_actions = {
              TranslatableString("action", "Full screen"),
              Checkable::Yes
              ),
-    UiAction("about",
+    UiAction("about-musescore",
              mu::context::UiCtxAny,
              mu::context::CTX_ANY,
-             TranslatableString("action", "&About…")
+             TranslatableString("action", "About &MuseScore…")
```
Comment (Jojo-Schmitz) [inline src/appshell/internal/applicationuiactions.cpp:60]: Leave the & at the A, ("&About MuseScore…"), it conflicts with "About &MusicXML…" otherwise

Comment (pratyaksh1610) [reply src/appshell/internal/applicationuiactions.cpp]: Done

**Comment by Tantacrul** [comment]
As a matter of interest, why would 'About MuseScore' go below 'About Qt' and 'About MusicXML' ?

Surely 'About MuseScore' is more important?

Also - just checked to confirm that this doesn't affect MacOS, which is good.

**Comment by pratyaksh1610** [comment]
@Tantacrul There was no particular reason except the order of strings in increasing order of lengths.
Surely `About MuseScore` is more important.

**Comment by Tantacrul** [comment]
I would generally arrange options like these in order of priority. I personally don't really see a need for either the Qt or the MusicXML popup.

For now, I think it's easiest to just put the MuseScore option on top of the three.

**Comment by cbjeukendrup** [comment]
The Qt popup is a requirement from Qt. About the MusicXML popup I don't know.

**Comment by pratyaksh1610** [comment]
Made the change as discussed.

**Review by cbjeukendrup — APPROVED** [review]
If @Tantacrul is happy, I'm happy too :)

**Comment by Jojo-Schmitz** [comment]
FWIW: I've been translating "&About…" into German as "&Über MuseScore…" ever since ;-)

**Comment by Tantacrul** [comment]
> If @Tantacrul is happy, I'm happy too :)

If About MuseScore is on top, then I'm over-the-moon. Merge city.

**Comment by pratyaksh1610** [comment]
@Tantacrul Yeah, "About MuseScore" is on the top now. 😆

**Comment by cbjeukendrup** [comment]
But now it's at the top of the whole menu. @Tantacrul Didn't you mean "at the top of the three _About_ items"?

**Comment by Tantacrul** [comment]
> But now it's at the top of the whole menu. @Tantacrul Didn't you mean "at the top of the three About items"?

Lol, yes. That's what I meant

**Comment by pratyaksh1610** [comment]
Looks like I misunderstood. Lol. 😆

**Comment by pratyaksh1610** [comment]
@Tantacrul Is it ready for merge ?

**Comment by Tantacrul** [comment]
I don't have time to check this again.

But if this is here, then yes 😄 
image

**Comment by pratyaksh1610** [comment]
> I don't have time to check this again.
> 
> But if this is here, then yes 😄 image

Yes it is at the correct position, I forgot to update the thread.

**Pull request merged**